docs/source/getting-started/tabular-data/index.rst

```diff
@@ -116,7 +120,11 @@ Context Parameters
   In this case, the influence is measured in terms of the number of rows an individual may contribute to your dataset. 
   Since we are analyzing quarterly data across 9 years, where an individual contributes up to one record per quarter,
   the unit of privacy corresponds to 36 row contributions. 
-  If we were to analyze a particular quarter in a particular year, the unit of privacy would be 1 since each individual would contribute at most one row. 
+  If we were to analyze a particular quarter in a particular year, the unit of privacy would be 1 since each individual would contribute at most one row.   
+  Alternatively, if you have a identifier column ``PIDENT``, 
+  you could set the privacy unit to ``dp.unit_of(contributions=1, identifier="PIDENT")``.
+  For simplicity, examples will express the privacy unit in terms of 36 row contributions, 
+  except when demonstrating how to use identifiers.
```

Comment:
To me, it seems simpler and less error-prone to use `identifier`, now that we have it.

Reply:
How about we keep things how they are for now, and maybe follow up with that later. IMO it has the same complexity, but in a different place (in the query). Analysts also may or may not have IDs.